### PR TITLE
nullrod flavor text

### LINKED ISSUE #19871: The obsidian rod is teeming with divine power. You feel like you could pulverize a horde of undead with this.
(WEB REPORT BY: mask2697 REMOTE: 172.93.109.202:7777)
# Revision
c656f910240c361da8a65ba15ae3bc448a5f5e33

# Description

This description applies regardless of what the null rod actually is
# Steps to Reproduce

get null rod, make it anything but null rod, wield it
# What you Expected

relevant name
# What Actually Happened

Dissapointment

### PR DESCRIPTION
**Pull request by RealestEstate**
Closes #19871.

### CODE REVIEW

**Comment by SGuivenne** [comment]
The bug your referencing didn't get properly referenced.

**Comment by RealestEstate** [comment]
how do I do proper?

**Comment by Kammerjunk** [comment]
`#19871` becomes #19871

**Comment by RealestEstate** [comment]
thankee

File: code/game/objects/items/weapons/null_rod.dm [hunk 1]
```diff
@@ -79,7 +80,7 @@
 /obj/item/weapon/nullrod/pickup(mob/living/user as mob)
 	if(user.mind)
 		if(user.mind.assigned_role == "Chaplain")
-			to_chat(user, "<span class='notice'>The obsidian rod is teeming with divine power. You feel like you could pulverize a horde of undead with this.</span>")
+			to_chat(user, "<span class='notice'>The [src] is teeming with divine power. You feel like you could [fluff_pickup] a horde of undead with this.</span>")
```
Comment (DamianX) [inline code/game/objects/items/weapons/null_rod.dm]: This will result in "The the nullrod is". Change it to `\The [src]`.

**Pull request merged**